robot_controllers_interface/src/controller_manager.cpp

```diff
@@ -162,14 +162,14 @@ int ControllerManager::requestStart(const std::string& name)
 int ControllerManager::requestStop(const std::string& name)
 {
   // Find controller
-  for (ControllerList::iterator c = controllers_.begin(); c != controllers_.end(); c++)
```

Comment:
If melodic only support c++11 or better this is okay, otherwise might need to leave old interator stuff.